src/derivatives/atlas.md

```diff
@@ -472,6 +473,141 @@ A guide for using macros can be found at
 })
 }}
 
+**Producing a new template AND atlas.**
+Atlasing is often performed with reference to a *custom* standard space.
+In this case, a feature template map is generated from all the participant(s)
+in the study, and the atlas' artifacts are produced with reference to that
+template.
+
+Either by generating the template space with aligning to a pre-existing template,
+or by estimating a transform between templates by means of image registration,
+a new template definition MUST be employed if the new template generates
+a new [*space*](../common-principles.md#definitions).
+For example, let's imagine that PS13 first generated a template nuclear imaging
+map and after that, a corresponding [*atlas*](../common-principles.md#definitions)
+was defined.
+In that case, the [`atlas-<label>`](../glossary.md#atlas-entities) SHOULD be omitted
+except several atlases need specification:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "ps13-pipeline": {
+      "tpl-PS13": {
+         "pet": {
+            "tpl-PS13_desc-nopvc_dseg.nii.gz": "",
+            "tpl-PS13_desc-pvc_dseg.nii.gz": "",
+            "tpl-PS13_dseg.json": "",
+            "tpl-PS13_dseg.tsv": "",
+            "tpl-PS13_stat-std_desc-fnirtNopvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-fnirtNopvc_mimap.nii.gz": "",
+            "tpl-PS13_stat-std_desc-fnirtPvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-fnirtPvc_mimap.nii.gz": "",
+            "tpl-PS13_stat-std_desc-nopvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-nopvc_mimap.nii.gz": "",
+            "tpl-PS13_stat-std_desc-pvc_mimap.json": "",
+            "tpl-PS13_stat-std_desc-pvc_mimap.nii.gz": "",
+         },
+      },
+   }
+})
+}}
+
+Let's complete the above example by adding two new atlases to the existing
+template and (*default*) atlas:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "ps13-with-atlases-pipeline": {
+      "tpl-PS13": {
+         "pet": {
+            "tpl-PS13_atlas-Economo1916_desc-nopvc_dseg.nii.gz": "",
```

Comment:
In which space is this? We are guessing this is in the PS13 space, but for generic users this might be hard to guess.
So for consistency it would be good to add space here everywhere as well (fsaverage and MNI152Lin).

Reply:
You are guessing what the example intends to express. Since the template *instantiates* the space, then space is not necessary anymore. Think of subject:

`sub-01_atlas-Economo1916_seg-thalamus_dseg.nii.gz` - tells you that this subject has been segmented using the Economo atlas, and this file contains discrete segment (or segments) corresponding to the Thalamus. It would not make sense to say:

`sub-01_atlas-Economo1916_space-01_seg-thalamus_dseg.nii.gz`, because regardless of what other space that atlas was brought in, it is now in the subject's native space.

Now, your second sentence makes me doubt whether that was the question:

> So for consistency it would be good to add space here everywhere as well (fsaverage and MNI152Lin).

In this case, fsaverage and MNI152NLin are intendedly removed (they are shown above anyway). Here, the idea was to show the case that a PS13 template was generated in a custom space (very much revolving around @pwighton's comment about whether you want your results to be *generative* or not of a new space, in this case showing the other part when you do want to generate a new space).

So `tpl-PS13_atlas-Economo1916_space-{fsaverage,MNI152Lin}_desc-nopvc_dseg.nii.gz` would mean a very different thing.

Also, for transparency and reproducibility, I would recommend people instead (or in addition) provide the transformations between the *new-space-generating* template and fsaverage and MNI152Lin.

I'm not sure adding those conversions would help clarify this example -- I'd prefer to keep it focused on what it is trying to show.